app/controllers/application_controller.rb

```diff
@@ -19,4 +21,18 @@ def admin_authenticated
   def require_user
     render file: '/public/404' unless current_user
   end
+
+  def store_location
+    return unless request.get?
```

Comment:
I haven't seen a return unless like this in Ruby before. What exactly is it doing?